rosdep/python.yaml

```
@@ -6496,6 +6496,16 @@ python3-pyqtgraph:
   fedora: [python3-pyqtgraph]
   gentoo: [dev-python/pyqtgraph]
   ubuntu: [python3-pyqtgraph]
+python3-pystemd:
+  debian: [python3-pystemd]
```

The `python3-pystemd` package is only available in Debian Bullseye, which is the current testing / upcoming stable distribution.

```suggestion
  debian:
    bullseye: [python3-pystemd]
```